sasl_bind.c

```diff
@@ -0,0 +1,21 @@
+#include <ldap.h>
+#include "sasl_bind.h"
+
+static int
+sasl_trivial_interact (LDAP * ld, unsigned flags, void *defaults,
+                       void *sasl_interact)
+{
+  (void) ld;
+  (void) flags;
+  (void) defaults;
+  (void) sasl_interact;
+  return LDAP_SUCCESS;
+}
+
+int
+trivial_external_sasl_bind (LDAP * ld)
+{
+  return ldap_sasl_interactive_bind_s (ld, NULL, "EXTERNAL", NULL, NULL,
```

Comment:
`olcAccess: {0}to * by dn.exact=gidNumber=123+uidNumber=456,cn=peercred,cn=external,cn=auth manage by * break`

This how it looks at server side.